### ISSUE TITLE
Can't open this app

### ISSUE THREAD

**Issue by zhiyongwu** [n0shake/Clocker#72]
Nothing happens after I click the Clocker icon.
Current version is 1.6.16 from mac app store.
Mac os version is macos mojave 10.14.6

**Comment by n0shake** [comment]
Yikes, that's bad. I'm waiting for my crash logs to come back. In the meanwhile, resetting your preferences should help you *atleast* launch the app. Sorry for the inconvenience.

1. Make sure Clocker is closed (force quit through Activity Monitor if necessary)
2. Open up the Terminal app and type in the command: `defaults delete com.abhishek.Clocker`
3. Hit the return key
4. Restart Clocker

**Comment by zhiyongwu** [comment]
This app works after I type the command and click the icon.
But when I change Menubar Mode from Compact to Standard in Preference > Appearance > Menubar Display Options, it crash and I can't launch it anymore. When I repeat this it happens again.

**Comment by n0shake** [comment]
Thanks for the repro steps. Can you share the following to help me debug:
- List of timezones you've added
- Are you using any other language than English?

**Comment by zhiyongwu** [comment]
-  I only add one timezone.It is Asia/Shanghai.

- Yes my system language is Chinese.

**Comment by zhiyongwu** [comment]
Is there any crash log in my computer.Maybe I can paste it here.

**Comment by n0shake** [comment]
Thanks but I've got the crash logs from my crash reporting service. 

This is because of a silly index out of bounds crash that might occur for users whose preferred first language is not English. I've fixed it via c0caf1ee8681282f024b92981d1fb3a136fc8d64. I'll test more and release this version over the weekend. In the meanwhile, I'll suggest to turn the **Include Days** option in menubar preferences to No.
